src/main/java/ch/uzh/ifi/hase/soprafs23/service/GameService.java

```diff
@@ -254,6 +254,9 @@ public void deletePlayerFromTeam(Team team, String playerName, int accessCode) {
             if (player.getName().equals(playerName)) {
                 team.getPlayers().remove(playerIdx);
                 gameRepository.flush();
+                if (team.getPlayers().size() <2) {
```

Comment:
What is the intuition behind this?

Reply:
to handle the logic in the backend, when a team has not enough players left.